data/resources/ui/content-chat-action-bar.ui

```diff
@@ -109,5 +109,31 @@
         </child>
       </object>
     </child>
+    <child>
+      <object class="GtkButton" id="unblock_button">
+        <property name="hexpand">True</property>
+        <property name="label">Unblock</property>
+        <style>
+          <class name="red-button"/>
+        </style>
+      </object>
+    </child>
+    <child>
+      <object class="GtkButton" id="mute_button">
+        <property name="hexpand">True</property>
+        <style>
+          <class name="blue-button"/>
+        </style>
+      </object>
+    </child>
+    <child>
+      <object class="GtkButton" id="join_button">
+        <property name="hexpand">True</property>
+        <property name="label">Join</property>
+        <style>
+          <class name="blue-button"/>
+        </style>
+      </object>
+    </child>
```

Comment:
It seems that these buttons can only be shown one at the time, right? I think that it makes sense to create a `GtkStack` that wraps everything inside the `GtkBox` and so creating multiple pages for each of the state. Also, this will make the buttons children of `GtkBox` + `.toolbar`, which will apply the correct style for them (6px margins + .flat).